.travis.yml

```diff
@@ -1,11 +1,13 @@
 language: ruby
 rvm:
   - 2.3
-# whitelist branches that can publish
-# to the repo
-branches:
-  only:
-  - master
```

Comment:
@ondreian LMK if you think we should remove this section now, or keep it as a transitional step but remove in a follow-up PR to avoid incidental publishing of non-master changes.